fastlane/metadata/android/ko-KR/title.txt

```diff
@@ -1 +0,0 @@
-워드프레스 – 웹사이트 제작 도구
```

Comment:
Title has changed to `waiting` status in GlotPress since last time — apparently to suggest keeping the brand name "WordPress" unchanged; and since those new suggestions haven't been approved and are still in waiting, I guess that's why this file was deleted here.

<img width="1189" alt="image" src="https://user-images.githubusercontent.com/216089/168326498-b0cc1c74-4f88-42d1-9ffa-9b60b56caf5e.png">